lib/constellation.rb

```diff
@@ -0,0 +1,39 @@
+class ConstellationHttpException < StandardError
+end
```

Comment:
Nah this is our handling of an exception from Constellation basically (a non sucessful response from that service).

Are you saying we should also rescue from `Net::HTTP` errors in a specific way?